packages/zendesk-adapter/src/change_validators/one_translation_per_locale.ts

```diff
@@ -90,20 +101,20 @@ export const oneTranslationPerLocaleValidator: ChangeValidator = async (changes,
         CATEGORY_TRANSLATION_TYPE_NAME,
         SECTION_TRANSLATION_TYPE_NAME].includes(instance.elemID.typeName))
 
-  // it is a set to avoid duplicate of parents from translations which have the same parent
-  const parentInstances = relevantInstances
+  const parentInstanceIds = _.uniqBy(relevantInstances
     .filter(instance => isTranslation(instance))
     .flatMap(getParents)
-    .map(translationParentInstance => translationParentInstance.value)
+    .flatMap(parent => (isReferenceExpression(parent) ? parent.elemID : undefined))
+    .filter(isDefined), id => id.getFullName())
```

Comment:
readability nit:
```suggestion
  const parentInstanceIds = _.uniqBy(
    relevantInstances
      .filter(instance => isTranslation(instance))
      .flatMap(getParents)
      .flatMap(parent => (isReferenceExpression(parent) ? parent.elemID : undefined))
      .filter(isDefined),
    id => id.getFullName()
  )
```
(the `id => id.getFullName()` part should be at the same "level" as `relevantInstances.<...>`)